data_summaries/Cidar.json

```diff
@@ -0,0 +1,32 @@
+{
+  "dataset-name (must match unique identifier)": {
+    "Unique Dataset Identifier": "cidar",
+    "Dataset Name": "cidar",
+    "Paper Title": "CIDAR: Culturally Relevant Instruction Dataset For Arabic",
+    "Dataset URL": "https://github.com/ARBML/CIDAR",
+    "GitHub URL": "https://github.com/ARBML/CIDAR",
+    "Hugging Face URL": "https://hf.co/datasets/arbml/CIDAR",
+    "Papers with Code URL": "https://paperswithcode.com/dataset/cidar",
+    "ArXiv URL": "https://arxiv.org/abs/2402.03177",
+    "Semantic Scholar Corpus ID": "",
```

Comment:
Corpus ID is: 267413026

Can be found on this page: https://www.semanticscholar.org/paper/CIDAR%3A-Culturally-Relevant-Instruction-Dataset-For-Alyafeai-Almubarak/8e4fff506c2ce70eeb770a63848209e351a39033